Index: test/Export/LateXDocTest.java
===================================================================
diff --git a/test/Export/LateXDocTest.java b/test/Export/LateXDocTest.java
--- a/test/Export/LateXDocTest.java	
+++ b/test/Export/LateXDocTest.java	
@@ -2,12 +2,24 @@
 
 import org.junit.jupiter.api.Test;
 
+import java.util.ArrayList;
+import java.util.List;
+
 import static org.junit.jupiter.api.Assertions.*;
 
 class LateXDocTest {
 
+    DocumentationNode bold=new DocumentationNode(false,"texte en bold",new ArrayList<String>(List.of("bd")));
+    DocumentationNode it=new DocumentationNode(false,"texte en italique",new ArrayList<>(List.of("it")));
+
+    DocumentationNode title=new DocumentationNode(false,"titre",new ArrayList<>(List.of("title")));
+
+    DocumentationNode bdit=new DocumentationNode(false,"texte en italique et bold",new ArrayList<>(List.of("it","bd")));
+    LateXDoc l=new LateXDoc();
     @Test
     void user() {
+        l.user(new ArrayList<>(List.of(it,bold)));
+        l.user(new ArrayList<>(List.of(bdit,title)));
     }
 
     @Test
